Caboodle/VersionTracking/VersionTracking.shared.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public static class VersionTracking
+    {
+        const string versionTrailKey = "VersionTracking.Trail";
+        const string versionsKey = "VersionTracking.Versions";
+        const string buildsKey = "VersionTracking.Builds";
+
+        static readonly Preferences preferences = new Preferences($"{AppInfo.PackageName}.caboodle");
+
+        static Dictionary<string, List<string>> versionTrail;
+
+        public static void Track()
```

Comment:
Should this really be required? Couldn't we just make a static constructor that gets called when anyone accesses a method?

I could see people calling IsFirstLaunchEver all the time...